app/src/main/java/com/spoony/spoony/presentation/placeDetail/PlaceDetailRoute.kt

```diff
@@ -226,13 +237,50 @@ private fun PlaceDetailScreen(
             isAddMap = isAddMap,
             onScoopButtonClick = onScoopButtonClick,
             onSearchMapClick = {
-                // 네이버 길찾기 코드
+                searchPlaceNaverMap(
+                    latitude = latitude,
+                    longitude = longitude,
+                    placeName = placeName,
+                    context = context
+                )
             },
             onAddMapButtonClick = onAddMapButtonClick
         )
     }
 }
 
+private fun searchPlaceNaverMap(
+    latitude: Double,
+    longitude: Double,
+    placeName: String,
+    context: Context
+) {
+    val url = "nmap://place?lat=$latitude&lng=$longitude&name=$placeName&appname=${context.packageName}"
+    val isInstalled = try {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
+            context.packageManager.getPackageInfo(
+                "com.nhn.android.nmap",
+                PackageManager.PackageInfoFlags.of(0)
+            )
+        } else {
+            context.packageManager.getPackageInfo("com.nhn.android.nmap", 0)
+        }
+        true
+    } catch (e: PackageManager.NameNotFoundException) {
+        false
+    }
+    if (!isInstalled) {
```

Comment:
```suggestion
    if (isInstalled) {
```
P4: 이렇게 ! 안붙은거 먼저 해주면 너무 좋을 것 같아요..!!